protocol/daemons/flags/flags.go

```diff
@@ -11,8 +15,9 @@ const (
 	// Flag names
 	FlagUnixSocketAddress = "unix-socket-address"
 
-	FlagPriceDaemonEnabled     = "price-daemon-enabled"
-	FlagPriceDaemonLoopDelayMs = "price-daemon-loop-delay-ms"
+	FlagPriceDaemonEnabled                = "price-daemon-enabled"
+	FlagPriceDaemonLoopDelayMs            = "price-daemon-loop-delay-ms"
+	FlagPriceDaemonExchangeConfigOverride = "price-daemon-exchange-config-override"
```

Comment:
Meta: should this be a file path to a json or toml file instead of the actual override strings?

Passing `"{\"exchange_query_configs\":[{\"exchange_id\":\"Bybit\",\"disabled\":true},{\"exchange_id\":\"Binance\",\"interval_ms\":100}]}"` feels a bit cumbersome (given all the \ you need to add) and can potentially be really long. 

I am leaning towards this being a filepath